source/frontend/src/components/maps/leaflet/Layers/util.tsx

```diff
@@ -272,10 +272,10 @@ export const isPimsBoundary = (
   return feature.id?.toString().startsWith('PIMS_PROPERTY_BOUNDARY_VW') ?? false;
 };
 
-export const isParcelMap = (
+export const isFaParcelMap = (
   feature: Supercluster.PointFeature<MarkerFeature>,
-): feature is Supercluster.PointFeature<PMBC_Feature_Properties> => {
-  return feature.id?.toString().startsWith('WHSE_CADASTRE.PMBC_PARCEL_FABRIC_POLY_SVW') ?? false;
+): feature is Supercluster.PointFeature<PMBC_FullyAttributed_Feature_Properties> => {
+  return feature.id?.toString().startsWith('WHSE_CADASTRE.PMBC_PARCEL_FABRIC_POLY_FA_SVW') ?? false;
 };
```

Comment:
Given that locally we don't have fully attributed parcel map this check would fail, right? Like **isFaParcelMap** will never be true in local development. Are there any implications that we should know about?